### DISCUSSION TITLE
can't show my log in static web site

### DISCUSSION THREAD

**Discussion by HongsCode** [vercel/next.js#49428] [Help]
### Summary

using AWS S3 static website
add to some log

next build && next export then upload AWS S3

but can't show my modify code. 
check to confirm chuck file, then it's exist

but when check on website, not show log.
why?

### Additional information

```js
"next": "12.2.5",
    "react": "18.2.0",
```

**Comment by icyJoseph** [comment]
Hi, are you using PWA?

**Reply by HongsCode** [reply]
what is PWA?

**Reply by HongsCode** [reply]
no, pure nextjs

**Reply by icyJoseph** [reply]
Aha, I meant Progressive Web App, typically it uses next-pwa. PWA can cause stale content to show.

Are you able to find the "modified" code in the `out` directory after `next export`?
